### PR TITLE
Remove LogTooLongException

### PR DESCRIPTION
**Pull request by omgapuppy**
Remove uncaught exception, write a test case, fail silently like other
libraries

### CODE REVIEW

**Comment by StephenHynes7** [comment]
@m0wfo thoughts? I am okay with this as a temp fix while we fix the logic for big stack traces.

File: src/main/java/com/logentries/net/AsyncLogger.java [hunk 1]
```diff
@@ -419,7 +419,7 @@ public void addLineToQueue(String line) {
 	}
 
 	private void addLineToQueue (String line, int limit) {
-		if (limit == 0) { throw new LogTooLongException(); }
+		if (limit == 0) { return; }
```
Comment (chrismlogentries) [inline src/main/java/com/logentries/net/AsyncLogger.java]: maybe add a `dbg()` statement saying 'message too long' or something

**Comment by chrismlogentries** [comment]
+1 when Travis is happy

**Comment by ardimehist-r7** [comment]
build failure seems to have something to do with the forked JVM not shutting down when using openjdk - don't think it's anything to do with my changes

**Comment by ardimehist-r7** [comment]
@m0wfo [this](https://github.com/logentries/le_java/pull/49/commits/e77b9067d4037a05833a5d257b0727bec85cf937) finally sorted the build failure

**Comment by m0wfo** [comment]
@ardimehist-r7 that is horrible but yey, I guess :)

**Pull request merged**